src/modules/transaction/hooks/useTransactionFee/useTransactionFee.js

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+import { useCommandSchema } from '@network/hooks/useCommandsSchema';
+import { useAuth } from '@auth/hooks/queries/useAuth';
+import { computeTransactionMinFee, getParamsSchema } from './utils';
+/**
+ *
+ * @param {object} data
+ * @param {boolean} data.isValid Whether the transaction is valid or not. TxComposer defines this
+ * @param {object} data.wallet The sender account info as returned by useDeprecatedAccount
+ * @param {object} data.transaction Transaction object as Lisk Element expects without fee
+ * @returns {object} The fee object with a total value, and a component value as an array of fees
+ * that contribute in the total value
```

Comment:
this is outdated 😅 the params are changed now

Reply:
Yes, wallet is changed to address, updated it.